lib/v2/components/user_info_card.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/v2/components/profile_avatar.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+
+class UserInfoCard extends StatelessWidget {
+  final String? imageUrl;
+  final String account;
+  final String? name;
+  final GestureTapCallback? resultCallBack;
+
```

Comment:
I thing `onTap` is a better name (denotes the action that triggers the callback) also we already know its a callback param.

Reply:
component was deleted.